java/connector-node/tracing/src/main/java/com/risingwave/tracing/TracingSlf4jAdapter.java

```diff
@@ -28,6 +28,42 @@ public class TracingSlf4jAdapter implements Logger {
 
     private final String name;
 
+    private void logIfEnabled(int level, String msg) {
+        if (TracingSlf4jImpl.isEnabled(level)) {
+            TracingSlf4jImpl.event(name, level, msg);
+        }
+    }
+
+    private void logIfEnabled(int level, String format, Object arg) {
+        if (TracingSlf4jImpl.isEnabled(level)) {
+            TracingSlf4jImpl.event(
+                    name, level, new ParameterizedMessage(format, arg).getFormattedMessage());
+        }
+    }
+
+    private void logIfEnabled(int level, String format, Object arg1, Object arg2) {
+        if (TracingSlf4jImpl.isEnabled(level)) {
+            TracingSlf4jImpl.event(
+                    name,
+                    level,
+                    new ParameterizedMessage(format, arg1, arg2).getFormattedMessage());
+        }
+    }
+
+    private void logIfEnabled(int level, String format, Object... arguments) {
+        if (TracingSlf4jImpl.isEnabled(level)) {
+            TracingSlf4jImpl.event(
+                    name, level, new ParameterizedMessage(format, arguments).getFormattedMessage());
+        }
+    }
+
+    private void logIfEnabled(int level, String msg, Throwable t) {
+        if (TracingSlf4jImpl.isEnabled(level)) {
+            String sStackTrace = ExceptionUtils.getStackTrace(t);
+            TracingSlf4jImpl.event(name, level, String.format("%s: %s", msg, sStackTrace));
```

Comment:
Can we use the method provided in [ParameterizedMessage](https://logging.apache.org/log4j/2.x/javadoc/log4j-api/org/apache/logging/log4j/message/ParameterizedMessage.html#%3Cinit%3E(java.lang.String,java.lang.Object%5B%5D,java.lang.Throwable)) to be consistent with the original behavior?